src/Sampler.cpp.Rt

```diff
@@ -21,7 +21,7 @@ int Sampler::initCSV(const char *name)
      f = fopen(name, "wt");
      output("Initializing %s\n",filename);
      assert( f != NULL );
-     fprintf(f,"Iteration,");
+     fprintf(f,"Iteration,X,Y,Z,");
```

Comment:
@shkodm Please make the comma at begining of written stuff, not at the end. otherwise you'll get always a comma at the end of every line.

For instance, if you'll print "Iterations,X,Y,Z" and then write ",Rho", and ",U.x,U,y,U.z", etc. you won't get the comma at the end of line.